Tests/AnyLintTests/StatisticsTests.swift

```diff
@@ -31,8 +28,10 @@
 
         let checkInfo2 = CheckInfo(id: "id2", hint: "hint2", severity: .warning)
         Statistics.shared.found(
-            violations: [Violation(checkInfo: checkInfo2), Violation(checkInfo: checkInfo2)],
-            in: CheckInfo(id: "id2", hint: "hint2", severity: .warning)
+            violations: [
+                CheckInfo(id: "id2", hint: "hint2", severity: .warning):
+                    [Violation(checkInfo: checkInfo2), Violation(checkInfo: checkInfo2)]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line collection literals should have trailing commas.](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5691372)